variable_selection/Bikes_kulprit.ipynb

```diff
@@ -43,11 +43,9 @@
    "metadata": {},
```

Comment:
<p>We have to change the end of the sentence because now we are using all the covariables.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Grupo-de-modelado-probabilista/BART/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>